contracts/fixed_price_rev1.1.scilla

```diff
@@ -474,8 +522,26 @@ procedure HandleFulfillOrder(
 
     (* On behalf of buyer, marketplace transfers the seller_profit to seller *)
     ZRC2TransferFrom payment_token_address buyer seller seller_profit
+
+    (* Paying out a commission if there is one *)
```

Comment:
This one should be safe to uncomment, right?

Reply:
Yes, this is updated